lib/json.sh

```diff
@@ -1,6 +1,10 @@
 #!/usr/bin/env bash
 
-JQ="$BP_DIR/lib/vendor/jq-$(get_os)"
+JQ="/usr/bin/jq"
+if ! test -f "$JQ"; then
+  curl -Ls https://github.com/stedolan/jq/releases/download/jq-1.5/jq-linux64 > "/usr/bin/jq" \
```

Comment:
`jq` is always available on all Heroku stack images as of heroku/stack-images/pull/174, meaning this codepath is unused for Heroku builds.

The reason this is needed to make the unit tests pass for this repo, is that the unit tests (unlike the Hatchet tests) use the Docker images, which in the case of Cedar-14 haven't been updated for some time (due to restrictions about distributing ESM updates), so the `heroku/cedar:14` docker image doesn't have heroku/stack-images/pull/174 so doesn't have jq.

Also `/usr/bin` can only be written to by root, so this line would fail should it ever be needed for builds on the Heroku platform. It seems that Dokku doesn't run builds as root either, and also doesn't have jq, which is why there were reports of failures in #789.

Other options for this implementation would be:
1. To assume jq always exists (since it does on Heroku), so remove this download and instead modify the tests to `apt-get update && apt-get install jq` on Cedar-14 only, [here](https://github.com/heroku/heroku-buildpack-nodejs/blob/dfe7058a2d79a33bce1bd56c66ccf2f88b342f82/makefile#L41)
2. To update the step to install into the buildpack directory (or build directory), and ensure that directory is on path

For reduced complexity I would lean towards (1). Dokku users will still need to update their base images to include `jq`, but at least the error message they get would be clearer ("jq not found" etc).

Reply:
> Dokku users will still need to update their base images to include `jq` (for parity with the Heroku stack images)

Hmm looks like they do base their images on the Heroku stack images:
https://github.com/gliderlabs/herokuish/blob/master/Dockerfile

...so perhaps it just needs a new herokuish release, or for Dokku users to update their install to a newer version?

Either way, this sounds like something Dokku users should file against dokku (needing jq/newer image) :-)